src/layer/vector/Polygon.js

```diff
@@ -84,6 +84,34 @@ L.Polygon = L.Polyline.extend({
 		return this._map.layerPointToLatLng(center);
 	},
 
+	// @method contains (latlng: LatLng): Boolean
+	// Returns `true` if `latlng` is inside the (multi)polygon.
```

Comment:
Surely, and probably a good place to mention that all computations are performed on projected points.